components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs

```diff
@@ -161,13 +160,12 @@ public class PublishedNodesJobConverter {
                         MessageType = legacyCliModel.MessageEncoding,
                         WriterGroupId = !string.IsNullOrEmpty(dataSetSourceBatches.First().Connection.Group)
                             ? $"{dataSetSourceBatches.First().Connection.Group}"
-                            : $"{dataSetSourceBatches.First().Connection.Endpoint.Url}_" +
-                                $"{new ConnectionIdentifier(dataSetSourceBatches.First().Connection)}",
+                            : $"{dataSetSourceBatches.First().Connection.Endpoint.Url}_{dataSetSourceBatches.First().Connection.CreateConsistentHash()}",
                         DataSetWriters = dataSetSourceBatches.Select(dataSetSource => new DataSetWriterModel {
                             DataSetWriterId = !string.IsNullOrEmpty(dataSetSource.Connection.Id)
                                 ? $"{dataSetSource.Connection.Id}"
-                                : $"{dataSetSource.Connection.Endpoint.Url}_" +
-                                    $"{dataSetSource.GetHashSafe()}",
+                                : $"{dataSetSource.SubscriptionSettings.PublishingInterval.GetValueOrDefault(legacyCliModel.DefaultPublishingInterval.GetValueOrDefault(TimeSpan.FromSeconds(1))).TotalMilliseconds}",
```

Comment:
Previous implementation was also using a consistent hash.

Reply:
for writer group I agree. for DataSetWriterId the GetHashSafe of the DataSetSource is dependent on the dataset content. here we want to avoid this dependency. I would tend to keep this change.